marathon/hbase-master.json

```diff
@@ -1,13 +1,18 @@
 {
     "id": "/cell-os/hbase-1/master", 
-    "cmd": "su hadoop -c \"curl -vvv -o /hbase/conf/hdfs-site.xml http://HDFS_NAMENODE:50070/conf && /hbase/bin/hbase master -Dhbase.master.info.port=$PORT0 -Dhbase.rootdir='hdfs://HDFS_NAMESPACE_ID/hbase-1' -Dhbase.zookeeper.quorum=ZK_QUORUM -D zookeeper.znode.parent=/hbase-1  start\"",
+    "cmd": "su hadoop -c \"/entrypoint.sh && curl -vvv -o /hbase/conf/hdfs-site.xml http://HDFS_NAMENODE_HOST/conf && /hbase/bin/hbase master -Dhbase.master.info.port=$PORT0 -Dhbase.rootdir='hdfs://HDFS_NAMESPACE_ID/hbase-1' -Dhbase.zookeeper.quorum=ZK_QUORUM -D zookeeper.znode.parent=/hbase-1  start\"",
     "container": {
         "docker": {
-            "image": "cellos/hbase:1.0.0"
+            "image": "cellos/hbase"
         },
         "type": "DOCKER",
         "volumes": []
     },
+    "env": {
+      "HBASE_HEAPSIZE": "3G",
+      "ZK_QUORUM": "ZK_QUORUM",
```

Comment:
If we want to have placeholders here, it's ok to mark them as such with something like `{{ZK_QUORUM}}`